### PR TITLE
Fix project warnings

### PR DESCRIPTION
**Pull request by progsmile**

### CODE REVIEW

File: apps/ehealth/lib/ehealth/cabinet/requests/user_search.ex [hunk 1]
```diff
@@ -3,11 +3,9 @@ defmodule EHealth.Cabinet.Requests.UserSearch do
 
   use Ecto.Schema
 
-  alias EHealth.Ecto.Base64
-
   @primary_key false
   embedded_schema do
-    field(:signed_content, Base64)
```
Comment (AlexKovalevych) [inline apps/ehealth/lib/ehealth/cabinet/requests/user_search.ex]: what warning does this fix?

File: apps/ehealth/lib/ehealth/guardian.ex [hunk 2]
```diff
@@ -22,4 +22,6 @@ defmodule EHealth.Guardian do
   end
 
   def build_claims(claims, _resource, _opts), do: {:ok, claims}
+
+  def resource_from_claims(_claims), do: {:error, :not_implemented}
```
Comment (AlexKovalevych) [inline apps/ehealth/lib/ehealth/guardian.ex]: @PavelVesnin what do you think?

Comment (PavelVesnin) [reply apps/ehealth/lib/ehealth/guardian.ex]: @AlexKovalevych warning with message "function is not implemented" is more preferable than this code, that's why I don't write it.

**Comment by progsmile** [comment]
You may merge

File: apps/ehealth/lib/ehealth/contracts/search.ex [hunk 3]
```diff
@@ -4,21 +4,18 @@ defmodule EHealth.Contracts.Search do
   use Ecto.Schema
   import Ecto.Changeset
 
-  alias EHealth.Ecto.StringLike
-  alias Ecto.UUID
-
   @primary_key false
   embedded_schema do
-    field(:id, UUID)
+    field(:id, Ecto.UUID)
```
Comment (AlexKovalevych) [inline apps/ehealth/lib/ehealth/contracts/search.ex]: Please revert

**Pull request merged**